Makefile

```diff
@@ -19,6 +19,8 @@ OPM_VERSION = v1.26.2
 # versions at https://github.com/kubernetes-sigs/kustomize/releases
 KUSTOMIZE_VERSION = v4.5.7
 
+SORT_IMPORTS_VERSION = v0.1.0
```

Comment:
There is a newer version. [v0.2.1](https://github.com/slintes/sort-imports/tags).